Model/Api/Request.php

```diff
@@ -28,30 +37,56 @@ public function __construct(
         if (!empty($params['requestType'])) {
             $this->requestType = $params['requestType'];
         }
+
+        if (class_exists(\Laminas\Http\Client::class)) {
+            $this->client = new \Laminas\Http\Client();
+        } elseif (class_exists(\GuzzleHttp\Client::class)) {
+            $this->client = new \GuzzleHttp\Client();
+        } elseif (class_exists(\Zend\Http\Client::class)) {
+            $this->client = new \Zend\Http\Client();
+        } else {
+            throw new Exception('No HTTP client library available.');
+        }
     }
 
     public function send($data)
     {
         $apiKey = $this->config->apiKey();
         $secret = $this->config->secret();
         $apiUrl = $this->env->apiUrl();
-
-        $client = new \Laminas\Http\Client();
-
-        $client->setUri("$apiUrl/{$this->path}?api_key=$apiKey");
-        $client->setOptions(['timeout' => $this->timeout]);
-        $client->setMethod($this->requestType);
-        $client->setRawBody(json_encode($data));
-        $client->setHeaders([
+        $headers = [
             'X-Authorization' => strval($secret),
-            'Content-Type'    => 'application/json'
-        ]);
+            'Content-Type' => 'application/json',
```

Comment:
could you please add alignment by assigned value to the hash. We use this type hash alignment in the code style.
```
'X-Authorization' => strval($secret),
'Content-Type'    => 'application/json'
```